data.sql

```diff
@@ -3,11 +3,36 @@ INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg)
         (2,'Gabumon', date '2018-11-15', 2, true, 8.00), 
         (3,'Pikachu', date '2021-01-07', 1, false, 15.04), 
         (4,'Devimon', date '2017-05-12', 5, true, 11.00);
-        
-INSERT INTO animals VALUES (5,'Charmander', '2020-02-08', 0, false, -11), 
-                           (6,'Plantmon', '2021-11-15', 2 , true , -5.7),
-                           (7,'Squirtle', '1993-04-02', 3 , false , -12.13),
-                           (8, 'Angemon', '2005-06-12', 1 , true , -45),
-                           (9,'Boarmon',  '2005-06-07', 7, true , 20.4),
-                           (10,'Blossom', '1998-10-13', 3 , true , 17),
-INSERT INTO animals VALUES (12,'Ditto', '2022-05-14' , 4 , true , 22);
\ No newline at end of file
+        (5,'Charmander', '2020-02-08', 0, false, -11), 
+        (6,'Plantmon', '2021-11-15', 2 , true , -5.7),
+        (7,'Squirtle', '1993-04-02', 3 , false , -12.13),
+        (8, 'Angemon', '2005-06-12', 1 , true , -45),
+        (9,'Boarmon',  '2005-06-07', 7, true , 20.4),
+        (10,'Blossom', '1998-10-13', 3 , true , 17),
+        (11,'Ditto', '2022-05-14' , 4 , true , 22);
+
+ALTER TABLE animals ADD PRIMARY KEY (id);
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id INT;
+ALTER TABLE animals ADD COLUMN owners_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
+ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY (owners_id) REFERENCES owners(id);
```

Comment:
These commands modify the database's schema, I would suggest you place them in `schema.sql` file which is the appropriate file for them.